public/content/wrapped-ether/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: What is Wrapped Ether (wETH)
```

Comment:
wETH is WETH?

Reply:
in apps iv seen WETH being more common personally

Reply:
WETH it is